Makefile

```diff
@@ -260,6 +260,23 @@ else
 	$(error addlicense must be installed for this rule: go get -u github.com/google/addlicense)
 endif
 
+### check_fmt: check formatting on files in repo
+check_fmt:
+ifeq ($(shell command -v goimports 2> /dev/null),)
+	$(error "goimports must be installed for this rule" && exit 1)
+endif
+ifeq ($(shell command -v addlicense 2> /dev/null),)
+	$(error "error addlicense must be installed for this rule: go get -u github.com/google/addlicense")
+endif
+	@{
+		if [[ $$(find . -name '*.go' -exec goimports -l {} \;) != "" ]]; then \
+			echo "Files not formatted; run 'make fmt'"; exit 1 ;\
+		fi ;\
+		if ! addlicense -c -f license_header.txt $$(find . -name '*.go'); then \
```

Comment:
I've removed license from one file and after `make check_fmt` I did not get error returned but instead I got broken license:
```
~/projects/devworkspace-operator makefile-check-format *10 !1 > git --no-pager diff                                                                            33s 14:06:52
diff --git a/internal/cluster/info.go b/internal/cluster/info.go
index 615b848..5c2bc22 100644
--- a/internal/cluster/info.go
+++ b/internal/cluster/info.go
@@ -1,15 +1,3 @@
-//
-// Copyright (c) 2019-2020 Red Hat, Inc.
-// This program and the accompanying materials are made
-// available under the terms of the Eclipse Public License 2.0
-// which is available at https://www.eclipse.org/legal/epl-2.0/
-//
-// SPDX-License-Identifier: EPL-2.0
-//
-// Contributors:
-//   Red Hat, Inc. - initial API and implementation
-//
-
 package cluster
 
 import (
~/projects/devworkspace-operator makefile-check-format *10 !1 > make check_fmt                                                                                     14:06:52
Unable to connect to the server: dial tcp: lookup api.cluster-devtools-4ba1.devtools-4ba1.example.opentlc.com on 127.0.1.1:53: no such host
~/projects/devworkspace-operator makefile-check-format *10 !1 > git --no-pager diff                                                                                14:07:39
diff --git a/internal/cluster/info.go b/internal/cluster/info.go
index 615b848..27e5463 100644
--- a/internal/cluster/info.go
+++ b/internal/cluster/info.go
@@ -1,14 +1,16 @@
+// Copyright 2020 -f
 //
-// Copyright (c) 2019-2020 Red Hat, Inc.
-// This program and the accompanying materials are made
-// available under the terms of the Eclipse Public License 2.0
-// which is available at https://www.eclipse.org/legal/epl-2.0/
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
 //
-// SPDX-License-Identifier: EPL-2.0
-//
-// Contributors:
-//   Red Hat, Inc. - initial API and implementation
+//      http://www.apache.org/licenses/LICENSE-2.0
 //
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
 
 package cluster
 
~/projects/devworkspace-operator makefile-check-format *10 !1 >      
```

Reply:
Heh. `-c is not a shortcut for -check`
![Screenshot_20201223_141253](https://user-images.githubusercontent.com/5887312/102995115-084a6100-4529-11eb-84be-c3832390f2a9.png)

Reply:
Added a fix please verify and merge with squash not to propagate fixup commit.

Reply:
:facepalm: bad habit of shortening args, thanks

Reply:
Tested and the changes do what they're supposed to. I did notice that the addlicense check seems not very good -- it will fail if the license isn't present (which is the real reason we want to use it) but changing a license once it's on the file will go undetected :shrug: